modules/gdscript/gdscript_editor.cpp

```diff
@@ -54,16 +54,17 @@ void GDScriptLanguage::get_string_delimiters(List<String> *p_delimiters) const {
 }
 Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const String &p_base_class_name) const {
 
+	bool th = EDITOR_DEF("text_editor/completion/add_type_hints", true);
```

Comment:
Android build fails here, but I guess the whole file should be in `#ifdef TOOLS_ENABLED`, and not just a handful of duplicate includes as done currently?

Reply:
This really needs I cleanup, but since I intend to refactor the completion code anyway, I'll fix it then.